src/api.cpp

```diff
@@ -1467,6 +1467,14 @@ cvk_create_command_queue(cl_context context, cl_device_id device,
         return nullptr;
     }
 
+    if (!(config.ignore_out_of_order_execution.set &&
```

Comment:
Why do we want to check whether it is set? I feel like just checking the value should be enough, but I might be missing something here.